<launch>
<!-- Listen for a rosbag or camera feed, and write image frames to a folder. -->

<!-- Usage: roslaunch image_conversion.launch name:="NAME" -->
<arg name="name"/>
<!-- RealSense monocular camera -->
<arg name="left_cam_topic" default="/camera/color/image_raw"/> 
<!-- NUance car stereo cameras -->
<!-- <arg name="left_cam_topic" default="/camera_array/cam0/image_raw"/> -->
<arg name="right_cam_topic" default="/camera_array/cam1/image_raw"/> 

<!-- Start the node. -->
<node pkg="setup_pkg" type="image_conversion_node.py" name="image_conversion_node" args="$(arg name) $(arg left_cam_topic) $(arg right_cam_topic)" output="screen"/>
</launch>